Readability.js

```diff
@@ -1316,6 +1316,61 @@ Readability.prototype = {
     return metadata;
   },
 
+  /**
+   * Find all <noscript> that located after <img> node, and contains exactly
+   * single <img> element. Once it found, this method will replace the previous <img>
+   * with <img> inside <noscript>, then finally remove the <noscript> tag. This is
+   * done because in some website (e.g. Medium), they use lazy load method like this.
+   *
+   * @param Element
+  **/
+  _unwrapNoscriptImages: function(doc) {
+    // First, find div which only contains single img element, then put it out.
+    var divs = doc.getElementsByTagName("div");
+    this._forEachNode(divs, function(div) {
+      if (div.children.length == 1 && div.children[0].tagName === "IMG") {
+        div.parentNode.replaceChild(div.children[0], div);
+      }
+    });
+
+    // Next find img without source, and remove it. This is done to
+    // prevent a placeholder img is replaced by img from noscript in next step.
+    var imgs = doc.getElementsByTagName("img");
+    this._forEachNode(imgs, function(img) {
+      var src = img.getAttribute("src") || "",
+        srcset = img.getAttribute("srcset") || "",
+        dataSrc = img.getAttribute("data-src") || "",
+        dataSrcset = img.getAttribute("data-srcset") || "";
+      
+      if (src === "" && srcset === "" && dataSrc === "" && dataSrcset === "") {
```

Comment:
Nit: as suggested above, please use individual declarations. You don't need the `|| ""` or the `=== ""` because all you care about is whether the attribute value is truthy or not.

Reply:
Fixed in [38bcb08](https://github.com/mozilla/readability/pull/588/commits/38bcb08e0c1c31f384d56c8b2c3098a07237dda1).